sample-etc_systemd.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Dynamic DNS Update Client
-After=network.target
+After=network-online.target
```

Comment:
```suggestion
After=network.target network-online.target
```

Reply:
Great suggestion, updated and pushed. I just now saw the button commit suggestion. Sorry.